src/pydata_sphinx_theme/__init__.py

```diff
@@ -508,7 +508,7 @@ def setup(app):
     here = Path(__file__).parent.resolve()
     theme_path = here / "theme" / "pydata_sphinx_theme"
 
-    app.add_html_theme("pydata_sphinx_theme", theme_path)
+    app.add_html_theme("pydata_sphinx_theme", os.fsdecode(theme_path))
```

Comment:
Using `fsdecode` here because that was used some lines below for ``app.config.templates_path.append(os.fsdecode(theme_path / "_templates"))`` as well (not fully sure what's the difference between that and calling `str(..)` on the path object, cc @pradyunsg)

Reply:
fsdecode is the correct way to get the path representation. See the second paragraph in https://www.python.org/dev/peps/pep-0519/#rationale. :)

I hope I don't need-snipe you into reading the whole PEP.

Reply:
I guess what surprises me is that it hasn't been built-in to pathlib in that case. e.g., Ì was expecting to be able to do `theme_path.fsdecode()`

Reply:
That would defeat the purpose tho, because you're supposed to be able to pass other types of objects to fsdecode too!

It's similar to the string protocol, except that it is for paths and you don't get a default implementation (i.e. passing a pandas object to str works, vs fspath/fsencode/fsdecode fails) . I guess we could use os.fspath here, but I think we want to guarantee a string here so fsdecode is the right thing.

Reply:
Should you then in theory be using fsdecode here as well? https://github.com/pradyunsg/furo/blob/d6b217f8bb021f137928aa30f1b41e829e87d73f/src/furo/__init__.py#L366

Reply:
The thing is that here you are 100% sure that you have a pathlib Path object (so not eg None), and wondering if for that case it matters if you do `str(path)` or `os.fsdecode(path)`?

Reply:
https://github.com/python/cpython/blob/7a822c92782ffda8fa32a4b30a95b9de7cc1b8e6/Lib/pathlib.py#L630

If you *know*, then, no. :)